README.md

```diff
@@ -18,16 +18,17 @@ background activity resumes normally.
 
 1. Ensure your Kubernetes is using `containerd` as the container runtime.
    - e.g. for minikube, add `--container-runtime containerd` to `minikube start` command:
-     `minikube start --kubernetes-version=v1.20.0 --container-runtime=containerd`
+     `minikube start --kubernetes-version=v1.20.7 --container-runtime=containerd`
    - Support for other container runtimes should be possible, eventually.
 1. Install knative as normal. Note: for local development, you will need to use an external container registry, i.e. Docker Hub, GCR, etc.
-1. Create a service account with TokenReview permissions so that the Freezer
-   Daemonset can validate tokens with the API server:
-   `kubectl -n knative-serving create serviceaccount freeze-tokenreview`
-1. Install the Freeze-Prozy webhook and daemonset:
+1. Add a label to your nodes with the appropriate runtime.
+   - e.g. for minikube and containerd, run `kubectl label nodes minikube knative.dev/container-runtime=containerd`
```

Comment:
Shame we can't default this so you don't have to specify anything if you're using containerd (since that's what we told people to use in the minikube start line and probably most common in prod these days). Maybe a future update.